migrations/versions/14842761654b_.py

```diff
@@ -0,0 +1,31 @@
+"""empty message
+
+Revision ID: 14842761654b
+Revises: ba778ef9c615
+Create Date: 2020-11-11 07:18:58.675778
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "14842761654b"
+down_revision = "ba778ef9c615"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column("project_custom_editors", "enabled")
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "project_custom_editors",
+        sa.Column("enabled", sa.BOOLEAN(), autoincrement=False, nullable=False),
```

Comment:
@ramyaragupathy I had problems to downgrade with the `nullable=False`. Do you recommend a better solution?